### ISSUE TITLE
Code theft

### ISSUE THREAD

**Issue by EthanZeigler** [AkiraMiyakoda/cppIncludeGuard#1]
You probably want to see this.~~Someone stole your work and put their name on the license.~~

https://github.com/CyanCor/vscode-cppincludeguards/commit/08ebfdfe8d760be8e881e20fec4144d24e5f512a

https://marketplace.visualstudio.com/items?itemName=cyancor.cppincludeguards

**Comment by TomRiedl** [comment]
Hi Ethan, hi Akira,

thank you for the note and a short clarification why this plugin existed.

Due to a very tough project deadline our team needed a modified version of this plugin (using hashes, automatically remove pragma once and the ability to update existing include guards).
Intentionally we started to fork the extension but then uploaded it as separate extension to speed up deployment for using it. This extension has an MIT license that makes it possible to do this.

We never wanted to offend someone or try to adorn ourselves with borrowed plumes, especially because we love to support the open source community.

I just deleted the existing extension and will soon provide our changes as pull request to your project.

I'd like to sincerely apologize for this and I'm looking forward supporting your very great extension.

Best regards
Tom
